### ISSUE TITLE
Add tidy support for colnames

### ISSUE THREAD

**Issue by magnusfurugard** [magnusfurugard/featuretoolsR#16]
When calling `tidy_feature_matrix` variable names become very non-R-like.

Clean variable names using regexes, something like:

```
tidynames <- function(df) {
  n <- tolower(names(df))
  tn <- gsub("[^A-z0-9]", "_", n)
  tn <- gsub("(_+?$)|(__+?)", "", tn)
  names(df) <- tn
  return(df)
}
```

**Comment by magnusfurugard** [comment]
Support added in ec9462164917f7de9f28acd2c8ef05fccd956109

**Comment by magnusfurugard** [comment]
Resolved in #17
